src/util/bip158.rs

```diff
@@ -355,7 +355,9 @@ impl<'a> GCSFilterWriter<'a> {
 
     /// Add some data to the filter
     pub fn add_element(&mut self, element: &[u8]) {
-        self.elements.insert(element.to_vec());
+        if element.len () > 0 {
```

Comment:
Space between `len` and `()` (cargo fmt? :D)
Using `if !elements.is_empty() {` is more clean, though.

Reply:
modified